src/parquet/util/test-common.h

```diff
@@ -183,6 +183,15 @@ void random_byte_array(int n, uint32_t seed, uint8_t *buf,
     buf += out[i].len;
   }
 }
+
+void random_levels(int n, uint32_t seed, int16_t max_level, int16_t* out) {
+  std::mt19937 gen(seed);
+  std::uniform_int_distribution<uint16_t> d(0, max_level);
+  for (int i = 0; i < n; ++i) {
+    out[i] = d(gen);
+  }
+}
```

Comment:
What about refactoring `random_number` to call `random_int` except for float and double? 

```
template <>
void random_number(int n, uint32_t seed, double* out) {
  random_real(n, seed, -INF, INF, out);
}
```